MonitorControl.xcodeproj/xcshareddata/xcschemes/MonitorControl.xcscheme

```diff
@@ -59,6 +59,13 @@
             ReferencedContainer = "container:MonitorControl.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "skip_read_priviledges"
+            value = "true"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+      </EnvironmentVariables>
```

Comment:
What does this do exactly (I'm not familiar)? Also I think there's a typo in `skip_read_priviledges`? Shouldn't it be `skip_read_privileges`?